### ISSUE TITLE
[1.10.2] Crash on placing RF bank in world

### ISSUE THREAD

**Issue by Cat-McCatface** [TeamCoS/Modular-Systems#132]
Forge: 2092
ModularSystems: 1.10.2-3.1.0
Bookshelf API: 1.10.2-3.3.0
[crash-2016-09-19_19.43.03-client.txt](https://github.com/TeamCoS/Modular-Systems/files/481409/crash-2016-09-19_19.43.03-client.txt)

**Comment by pauljoda** [comment]
Yes I made a change to Bookshelf recently that I have yet to update on this mod. Sorry about that
